spec/lib/alaveteli_geoip_spec.rb

```diff
@@ -0,0 +1,123 @@
+# -*- encoding : utf-8 -*-
+require File.expand_path(File.dirname(__FILE__) + '/../spec_helper')
+
+describe AlaveteliGeoIP do
+
+	describe '.country_code_from_ip' do
+
+    it 'delegates to an instance of the class' do
+      instance = double
+      allow(AlaveteliGeoIP).to receive(:instance).and_return(instance)
+      allow(instance).to receive(:country_code_from_ip).with('127.0.0.1')
```

Comment:
Should this be an `expect`? Not sure if this is just a syntax change for new rspec?